Feat/screen services

Ismqt opened the pull request:
- Adición de Pantalla Pedidos
- Adición de Pantalla Detalle Producto
- Actualización del Mock de productos (productService)
- Conexión de los pedidos del Carrusel a la Pantalla Detalle Producto
- Acualización del Header

Ismqt committed:
- Adición de Pantalla Pedidos
- Adición de Pantalla Detalle Producto
- Actualización del Mock de productos (productService)
- Conexión de los pedidos del Carrusel a la Pantalla Detalle Producto
- Acualización del Header

Ismqt committed:
feat: agregando enlaces a servicios de productos

Albert2707 commented on src/components/ProductDetail.jsx:
```diff
@@ -0,0 +1,246 @@
+      setLoading(false)
+    }
+
+    console.log("[Debug] Available mock product IDs:", [
```
Borrar este console.log

Albert2707 reviewed (changes requested):
Eliminar console.log innecesarios.

Uriel020 commented on src/components/ProductDetail.jsx:
```diff
@@ -0,0 +1,246 @@
+      })
+    } else {
+      navigator.clipboard.writeText(window.location.href)
+      alert("Enlace copiado al portapapeles")
```
Eliminar el alert, eso crea aberturas en seguridad.

Utilizar sweet alert, pero en este caso como es una notificación mínima o se elimina o se agrega un ícono momentáneo.

Uriel020 commented on src/components/ProductDetail.jsx:
```diff
@@ -0,0 +1,246 @@
+
+  const handleAddToCart = () => {
+    if (product) {
+      alert(`${product.name} (x${quantity}) agregado al carrito.`)
```
Eliminar el alert, eso crea aberturas en seguridad.

Usar sweet alert.

Uriel020 commented on src/components/ProductDetail.jsx:
```diff
@@ -0,0 +1,246 @@
+
+  const handleBuyNow = () => {
+    if (product) {
+      alert(`Comprando ${product.name} (x${quantity})`)
```
Eliminar el alert, eso crea aberturas en seguridad.

Usar sweet alert.

Uriel020 reviewed (changes requested):
Sin contexto no pasa. Se está comprometiendo el app.jsx para el consumo del api directo.

Ismqt commented:
> Sin contexto no pasa. Se está comprometiendo el app.jsx para el consumo del api directo.

Elimino la función completa o que cambio me sugieres?

Uriel020 commented:
> > Sin contexto no pasa. Se está comprometiendo el app.jsx para el consumo del api directo.
> 
> Elimino la función completa o que cambio me sugieres?

Tienes que crear un contexto para los pedidos que maneje todas las funciones que solicitan al api, guíate del contexto de @WilsonCandelario095 que como no se han creado los clientes de esas apis, él puso bocetos de los mismos.

Ismqt commented:
> > > Sin contexto no pasa. Se está comprometiendo el app.jsx para el consumo del api directo.
> > 
> > 
> > Elimino la función completa o que cambio me sugieres?
> 
> Tienes que crear un contexto para los pedidos que maneje todas las funciones que solicitan al api, guíate del contexto de @WilsonCandelario095 que como no se han creado los clientes de esas apis, él puso bocetos de los mismos.

Entendido, on it, justo ando modificando par de cositas por los context, haré eso de paso

Ismqt committed:
- Eliminación de Logs
- Sustitución de Altert
- Optimización de Recursos en Pedidos
- Cambio de Diseño en pedidos
- Reparación de Filtro de Pedidos
- Cambio UX Carrito
- Adición del context de Pedidos

Ismqt committed:
Reparación de promos uwu

Ismqt committed:
Sacó D utils

Uriel020 reviewed (approved):
<img width="140" height="140" alt="image" src="https://github.com/user-attachments/assets/7139a0c4-03e7-47a1-a935-e9a61e2ec228" />

Albert2707 commented:
Approved
